x/config/config.go

```diff
@@ -0,0 +1,117 @@
+package config
```

Comment:
@breerly It does make sense, I just didn't find this information in readme about  experimentation process.